libqtile/utils.py

```diff
@@ -523,6 +537,29 @@ async def add_signal_receiver(
         return False
 
 
+async def remove_signal_receiver(bus: MessageBus, rule: str) -> bool:
```

Comment:
We can probably delete this function entirely. Don't think anything is using it.